sunpy/visualization/animator/image.py

```diff
@@ -173,7 +173,9 @@ def __init__(self, data, wcs, image_axes=[-1, -2], unit_x_axis=None, unit_y_axis
         self.slices_wcsaxes = list_slices_wcsaxes[::-1]
         self.unit_x_axis = unit_x_axis
         self.unit_y_axis = unit_y_axis
-        super().__init__(data, image_axes=image_axes, axis_ranges=axis_ranges, **kwargs)
+
+        # For some reason using `super()` here causes an error.
+        ImageAnimator.__init__(self, data, image_axes=image_axes, axis_ranges=axis_ranges, **kwargs)
```

Comment:
I don't expect you to know the answer since you say "for some reason" and "wtaf", but what's the deal?

Reply:
I have no flaming idea, the error being raised was:
```
  File "/home/circleci/project/.tox/build_docs/lib/python3.7/site-packages/sunpy/visualization/animator/image.py", line 176, in __init__
    super().__init__(data, image_axes=image_axes, axis_ranges=axis_ranges, **kwargs)
TypeError: super(type, obj): obj must be an instance or subtype of type
```

Reply:
Is this because of a funny interaction with the `@deprecated` dectorator? If you remove the dectorator does `super()` work?

Reply:
You are correct, good spot! I have updated the comment.